### PR TITLE
Fix named queries with reactive panache

### PR DESCRIPTION
**Pull request by gilvansfilho**
@loicmathieu Here are fix to named queries with reactive panache.

Tests are skipped and I can't realize why. Could you help me with this? 
I am using this command `.\mvnw test -f .\integration-tests\hibernate-reactive-panache\` and having this output [maven.log](https://github.com/quarkusio/quarkus/files/7464756/maven.log)

### CODE REVIEW

**Comment by geoand** [comment]
If you execute `mvn test -f integration-tests/hibernate-reactive-panache -Dstart-containers -Dtest-containers` you should be able to run the tests locally

**Comment by loicmathieu** [comment]
@gilvansfilho this looks promising, as soon as you successfully launch the tests and mark this PR as ready I'll do a closer review.

File: docs/src/main/asciidoc/hibernate-reactive-panache.adoc [hunk 1]
```diff
@@ -502,6 +507,18 @@ public class Person extends PanacheEntity {
...
+        return count("#Person.countByStatus", Parameters.with("status", status).map());
+    }
+
+    public static Uni<long> updateStatusById(Status status, long id) {
+        return update("#Person.updateStatusById", Parameters.with("status", status).and("id", id));
+    }
+
+    public static Uni<long> deleteById(long id) {
+        return delete("#Person.deleteById", id);
+    }
```
Comment (loicmathieu) [inline docs/src/main/asciidoc/hibernate-reactive-panache.adoc:521]:
```suggestion
    public static Uni<Long> countByStatus(Status status) {
        return count("#Person.countByStatus", Parameters.with("status", status).map());
    }

    public static Uni<Long> updateStatusById(Status status, Long id) {
        return update("#Person.updateStatusById", Parameters.with("status", status).and("id", id));
    }

    public static Uni<Long> deleteById(Long id) {
        return delete("#Person.deleteById", id);
    }
```

**Review by loicmathieu — CHANGES_REQUESTED** [review]

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks a lot !

**Pull request merged**